src/connection.ts

```diff
@@ -93,6 +91,22 @@ export interface ConnectionOpts {
    * When omitted, use a timeout of 30 seconds.
    */
   getExpiry?: (destination: string) => Date,
+  /**
+   * Callback for the consumer to perform accounting and choose to fulfill an incoming ILP Prepare,
+   * given the identifier of the connection, sequence of the packet, and amount received.
+   *
+   * If the returned Promise is resolved, the ILP Prepare will be fulfilled; if it is rejected,
+   * the ILP Prepare will be rejected. The ILP Fulfill will be immediately sent back after
+   * the Promise is resolved.
+   *
+   * Notes:
+   * - If `shouldFulfill` is provided, all connections MUST have a `connectionTag`, otherwise
```

Comment:
can we just make the connectionTag optional? I would be confused by that behavior as a user of the API unless I had carefully read the docs